boardgamegeek/api.py

```diff
@@ -888,17 +888,23 @@ def game(self, name=None, game_id=None, choose=BGGChoose.FIRST, versions=False,
         while added_items and len(game.comments) < total:
             page += 1
 
+            params['page'] = page
             xml_root = request_and_parse_xml(self.requests_session,
                                              self._thing_api_url,
-                                             params={"id": game_id,
-                                                     "pagesize": 100,
-                                                     "comments": 1,
-                                                     "page": page})
+                                             params=params,
+                                             timeout=self._timeout,
+                                             retries=self._retries,
+                                             retry_delay=self._retry_delay)
+
+            xml_root = xml_root.find("item")
+            if xml_root is None:
```

Comment:
Not sure if this error check is necessary here, but it seemed like a good idea to mimic the above error checking.